tests/ui/traits/item-privacy.stderr

```diff
@@ -8,9 +8,9 @@ LL |     S.a();
    |       ^
    |
    = help: items from traits can only be used if the trait is implemented and in scope
-help: trait `A` which provides `a` is implemented but not in scope; perhaps you want to import it
+help: trait `A` which provides `a` is implemented but not in scope
```

Comment:
Also unrelated, but would improve the quality of this and the other errors like this, but how difficult would it be to make this say:

```
help: trait `method::A` which provides `a` is implemented but not in scope
```

without the extra lines?